backoffice/config/urls.py

```diff
@@ -1,4 +1,4 @@
-from allauth.socialaccount.providers.orcid.views import oauth2_login
+from allauth.socialaccount.providers.orcid.views import oauth2_login, oauth2_callback
```

Comment:
I believe that these changes need to be reverted.

This is because.
for the sandbox orcid the route is indeed `api/oauth/authorized/orcid/`
however for the orcid we have in QA and will be in prod, the route is the default one (`/accounts/orcid/login/callback/`) so no need for this. 

As an alternative we can make it so that `api/oauth/authorized/orcid/` is also a valid route in our orcid in qa @drjova ?

Reply:
yes I will request adding / or verify that on orcid production as well. I will send an email

Reply:
So we use prod ORCID in QA and PROD and sandbox only in local?
It will take long to get a reply, right? Otherwise maybe I should make this route dynamic depending on the environment.

Reply:
@drjova could you also check that the prod environment is one of the allowed domains in the orcid prod app?
`https://backoffice.inspirehep.net`  `/api/oauth/authorized/orcid/` or `/accounts/orcid/login/callback/`